sql-statements/sql-statement-flashback-cluster-to-timestamp.md

```diff
@@ -0,0 +1,93 @@
+---
+title: FLASHBACK CLUSTER TO TIMESTAMP
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-flashback-cluster-to-timestamp/', '/docs-cn/dev/reference/sql/statements/flashback-cluster-to-timestamp/']
+---
+
+# FLASHBACK CLUSTER
+
+在 TiDB 6.4 中，引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
+
+## 语法
+
+{{< copyable "sql" >}}
+
+```sql
+FLASHBACK CLUSTER TO TIMESATMP '2022-09-21 16:02:50';
+```
+
+### 语法图
+
+```ebnf+diagram
+FlashbackClusterStmt ::=
+    "FLASHBACK" "CLUSTER" "TO" "TIMESTAMP" stringLit
+```
+
+## 注意事项
+
+* `FLASHBACK` 指定的时间点需要在 Garbage Collection (GC) life time 时间内，可以使用系统变量 `tidb_gc_life_time` 配置数据的历史版本的保留时间（默认值是 10m0s）。可以使用以下 SQL 语句查询当前的 safePoint，即 GC 已经清理到的时间点：
+
+```sql
+SELECT * FROM mysql.tidb WHERE variable_name = 'tikv_gc_safe_point';
+```
+
+* 仅支持拥有 `SUPER` 权限的用户执行 `FLASHBACK CLUSTER` 操作。
+* 在 `FLASHBACK CLUSTER` 指定的时间点到开始执行的时间段内不能存在非 `FLASHBACK CLUSTER` 类型的 DDL 记录。
+* 在执行 `FLASHBACK CLUSTER` 期间，TiDB 会主动断开所有非 `FLASHBACK CLUSTER` 的链接，并在禁止对相关字段进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
+* `FLASHBACK CLUSTER` 命令不能取消，一旦开始执行会一直重试，直到成功。
```

Comment:
Could we try to add some descriptions on the observability of the process, for example, how to check the progress of flashback even this is not implemented yet?